src/components/Competitions/Competitions.module.css

```diff
@@ -31,12 +31,73 @@
   padding-bottom: 10px;
 }
 
+.TopBoxParticipantsHeader {
+  font-weight: bold;
+  justify-content: center;
+  border-bottom-width: 100%;
+  border-bottom-color: black;
+  border-bottom-style: solid;
+}
+.TopBoxParticipantsText {
+  columns: 4 auto;
+  width: 100%;
+  display: center;
+}
+
 .AboutBox {
-  text-align: left;
+  text-align: justify;
+  text-align-last: left;
   word-wrap: break-word;
+  width: 52%;
+  display: inline-block;
 }
 
 .ImageCarousel {
   margin-top: 100px;
   margin-bottom: 100px;
 }
+.ReturnButton {
+  border-radius: 25px;
+  width: 5.5%;
+  height: auto;
+  background-color: #7a7a7a;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  text-transform: capitalize;
+  transition: 0.3s;
+  font-size: 75%;
+  position: sticky;
+  position: -webkit-sticky;
+  top: 3%;
+  z-index: 1000;
+  margin-left: 15%;
+}
+
+@media only screen and (max-width: 768px) {
+  [class*='ReturnButton'] {
+    width: 17%;
+    margin-left: 10%;
+    top: 3.6%;
+  }
+  [class*='TopBoxImage'] {
```

Comment:
aldri sett før. funker jo sikkert siden du har gjort det. men du kan også bare skrive `.TopBoxImage {...}`.

Punktum før navnet betyr at det gjelder for en klasse